src/main.js

```diff
@@ -95,6 +96,26 @@ function getNLSConfiguration() {
 	return { locale: initialLocale, availableLanguages: {} };
 }
 
+function migrateUserDataDir(newDir) {
+	var oldDir = path.join(process.env.HOME, '.config', product.nameShort);
```

Comment:
Why do you need to compute the old dir? Is it not much safer to use app.getPath('userData')